### ISSUE TITLE
systemd-nspawn crashing in 835.8.0

### ISSUE THREAD

**Issue by pquerna** [coreos/bugs#1010]
core dump:
http://people.apache.org/~pquerna/core.systemd-nspawn.0.8498f1635a0a4ac8b68e965931d9fea1.4432.1449102218000000.xz

not much info when debugging in toolbox:
https://gist.github.com/pquerna/c0a5a39c4bc159e039cf/a82c0794de4048526c20772ee95165c2f24031e4

**Comment by crawford** [comment]
What arguments are you passing to nspawn?

**Comment by pquerna** [comment]
in a unit file:

```sh
[Service]
ExecStart='/usr/bin/systemd-nspawn' '-D' '/opt/sft/versions/scaleft-platform-0.13.4/containers/sft-frontend-0.13.4/rootfs' '--machine' 'sft-dashboard.ip-10-0-1-25.us-west-2.compute.internal' '--tmpfs' '/tmp' '--capability=CAP_NET_BIND_SERVICE' '--bind-ro' '/opt/sft/conf/secrets/tls-client:/opt/sft/conf/secrets/tls-client' '/usr/bin/node' '/usr/src/app/bin/dashboard.js'
Restart=always
RestartSec=10s
```

**Comment by pquerna** [comment]
@crawford https://github.com/systemd/systemd/issues/1306 looks potentially to be the same issue, reported as fixed in v225, but no additional details on exact fix.

**Comment by pquerna** [comment]
Changing' ` --tmpfs' '/tmp' ` to `'--tmpfs' '/tmp:mode=0755'`` (passing in the mode) prevents the crash, and systemd-nspawn seems to work.

this seems like a relevant commit upstream, but hard for me to test this anywhere:

https://github.com/systemd/systemd/commit/a19222e1d39e8f0ae254bf02b7babb0bd73ee5be#diff-db6b16c9ade7e951e3bb793bec110098

**Comment by vcaputo** [comment]
Looks like this was fixed in v226 as a side-effect of f757855e: https://github.com/systemd/systemd/commit/f757855e#diff-db6b16c9ade7e951e3bb793bec110098R513
